src/int.cr

```diff
@@ -508,7 +508,7 @@ struct Int
     i = self ^ self
```

Comment:
Hmm. This returns 0 for all values, right? `self.class.zero` might be more obvious.